snippets/code_to_review3.R

```diff
@@ -0,0 +1,55 @@
+library(tidyverse)
+library(lubridate)
+library(ggplot2)
+library(maps)
+library(patchwork)
+
+#' This is script with analysis of volcanos for my
+#' geological article.
+
+volcano <- readr::read_csv('https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-05-12/volcano.csv')
+
+processed_volcano <- volcano %>%
+  mutate(logpop10 = log10(population_within_10_km+0.1)) %>%
+  select(primary_volcano_type, pop=logpop10, longitude, latitude)
+
+processed_volcano$primary_volcano_type <-
+  recode(processed_volcano$primary_volcano_type, Stratovolcano="Stratovolcano(es)")
+
+processed_volcano$primary_volcano_type <-
+  recode(processed_volcano$primary_volcano_type, `Shield(s)`="Shield")
+
+volcano_types <- processed_volcano %>% group_by(primary_volcano_type) %>%
+  summarise(count = n()) %>%
+  arrange(desc(count)) %>% head(8)
+
+processed_volcano$primary_volcano_type <- processed_volcano$primary_volcano_type %>%
```

Comment:
would it make sense to move this below to two calls to `recode`above and add a comment that explains why primary volcano types are modified ?

Reply:
agreed